### DISCUSSION TITLE
Decide on a naming standard for event objects

### DISCUSSION THREAD

**Discussion by abandonedaccount6235** [SkriptLang/Skript#4993] [General]
I think we should add a naming standard for how we name event objects. Some contributors use `e` and others use `event`. I think we should probably find a standard before this inconsistency continues to grow in the codebase. 

Related PR: #4992

**Comment by abandonedaccount6235** [comment]
Copying existing discussion from #4992:
![image](https://user-images.githubusercontent.com/28607612/181936816-5dc03ed1-bfe2-48fe-beb6-673e6b9e6048.png)
![image](https://user-images.githubusercontent.com/28607612/181936822-4b47ad8d-728d-431f-ba63-d46beda84636.png)

**Comment by Moderocky** [comment]
Just following up on my original comment, I'm in favour of `event` (and avoiding one-letter names for complex types in general.) \
I explain my reasoning fully in my style guide [here](https://github.com/Moderocky/Stylezie#local-variable-names) but the quick version is:
- Most people use single-letters for primitives (int i, boolean b, char c) and it helps not to get confused.
- You can always know what the variable is when reading a large code block e.g. on GitHub.
- Giving a parameter a good name is like a mini-documentation suggestion of what it's supposed to do.

**Comment by TheLimeGlass** [comment]
Should be `event`

You should always be using proper naming and not short cutting.
Developers can understand what `event` is without knowing the object is Event.

As Elon Musk tells SpaceX all the time "Stop, using, abbreviations! It makes it harder for the newer intern to understand."

**Comment by Itzdlg** [comment]
Don’t know if I should upvote-react a similar answer instead, as I don’t know how this’ll be decided, but I agree with Kenzie. Using `event` is proper & more verbose, especially when there could be any number of types for `e` like expression or exception.
